message_metrics/base/deploymentconfig.yaml

```diff
@@ -0,0 +1,155 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: message-metrics
+spec:
+  replicas: 1
+  selector:
+    service: message-metrics
+    app.kubernetes.io/name: thoth
+    app.kubernetes.io/component: investigator
+    app.kubernetes.io/managed-by: aicoe-thoth-devops
+  template:
+    metadata:
+      labels:
+        service: message-metrics
+        app.kubernetes.io/name: thoth
+        app.kubernetes.io/component: investigator
+        app.kubernetes.io/managed-by: aicoe-thoth-devops
+    spec:
+      serviceAccountName: investigator
+      containers:
+        - image: investigator:test
```

Comment:
here should be only `investigator` as this is `/base`

Reply:
++